syntax = "proto3";

package rules;

message CSMeanReversionRequest {
    string symbol = 1;
    int32 horizon = 2;
    bool use_attenuation = 3;
    float scaling_factor = 4;
    string scaling_type = 5;

}

message CSMeanReversionResponse {
    bytes series = 1;
}

service CSMeanReversion{
    rpc get_cs_mean_reversion (CSMeanReversionRequest) returns (CSMeanReversionResponse);
}